### PR TITLE
feat(gui): localized with crowdin

### PR DESCRIPTION
**Pull request by Misaka-L**
NOTE: it shouldn't be merged directly since it need a crowdin project ID (https://github.com/Misaka-L/vrc-get/blob/b2901d13b729a0e0f71706930312d9ce2374b957/vrc-get-gui/public/crowdin-in-context.js#L2)

#948

### CODE REVIEW

**Comment by anatawa12** [comment]
To avoid merging by mistake, I make this pr as draft.

**Review by anatawa12 — CHANGES_REQUESTED** [review]
Thank you for suggestion.

However, I think it's important to keep author information in git history and as far as I found it couldn't with crowdin.
Unless we found some solution for git author problem, I don't like to use crowdin.

https://github.com/vrc-get/vrc-get/issues/948#issuecomment-2106120758

File: vrc-get-gui/locales/de.json [hunk 1]
```diff
@@ -202,6 +200,6 @@
     "projects:toast:backup succeeded": "Backup erfolgreich erstellt",
     "settings:language": "Sprache",
     "settings:report issue": "Problembehandlung",
-    "settings:button:open issue": "Fehler melden",
-  },
+    "settings:button:open issue": "Fehler melden"
+  }
```
[the remark this answers is not in the document]
Comment (Misaka-L) [reply vrc-get-gui/locales/de.json:204]: solved by using (https://store.crowdin.com/jsonc)

[the remark this answers is not in the document]
Comment (Misaka-L) [reply vrc-get-gui/locales/de.json:204]: maybe need some work to let `json5` work in crowdin.

File: vrc-get-gui/src/commands.rs [hunk 2]
```diff
@@ -234,7 +234,7 @@ pub(crate) fn startup(app: &mut App) {
             } else if cfg!(windows) {
                 url.scheme() == "https" && url.host_str() == Some("tauri.localhost")
             } else {
-                url.scheme() == "tauri"
+                url.scheme() == "tauri" || (url.scheme() == "https" && url.host_str() == Some("accounts.crowdin.com"))
```
[the remark this answers is not in the document]
Comment (Misaka-L) [reply vrc-get-gui/src/commands.rs]: solved in https://github.com/vrc-get/vrc-get/pull/949/commits/e869bf4c5ae6cc472c45be5dd59aa49e60a15b94

File: vrc-get-gui/locales/de.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Misaka-L) [reply vrc-get-gui/locales/de.json:204]: I think it is possible to download the `json` file from Crowdin and convert it into a `json5` file using a GitHub Action workflow. Additionally, we can create a workflow that converts the source file from `json5` to `json` and then uploads it to Crowdin.

[the remark this answers is not in the document]
Comment (anatawa12) [reply vrc-get-gui/locales/de.json:204]: umm... is there any format without trailing comma problem that crowdin supports? changing format might be better fix

[the remark this answers is not in the document]
Comment (Misaka-L) [reply vrc-get-gui/locales/de.json:204]: > umm... is there any format without trailing comma problem that crowdin supports? changing format might be better fix

maybe [toml](https://store.crowdin.com/toml) or [yaml](https://store.crowdin.com/yaml) even xml?

**Comment by anatawa12** [comment]
https://github.com/vrc-get/vrc-get/issues/948#issuecomment-2106460897

**Pull request closed without merging**